Formula/vc.rb

```diff
@@ -38,7 +38,9 @@ def install
          return 0;
        }
     EOS
-    system ENV.cc, "test.cpp", "-std=c++11", "-L#{lib}", "-lvc", "-o", "test"
+    extra_flags = []
+    extra_flags += ["-lm", "-lstdc++"] unless OS.mac?
+    system ENV.cc, "test.cpp", "-std=c++11", "-L#{lib}", "-lvc", *extra_flags, "-o", "test"
```

Comment:
Maybe capitalization? Just guessing, so will need to check:
```suggestion
    system ENV.cc, "test.cpp", "-std=c++11", "-L#{lib}", "-lVc", *extra_flags, "-o", "test"
```